docs/docusaurus.config.js

```diff
@@ -98,6 +98,11 @@ const config = {
             label: "Applications",
             position: "left",
           },
+          {
+            href: "pathname:///python-sdk/api-reference-docs/",
```

Comment:
Also `pathname://` here is required in order for the sphinx docs to be properly rendered without having to refresh the page. For more on this: https://docusaurus.io/docs/advanced/routing#escaping-from-spa-redirects